test/api/test_tools.py

```diff
@@ -456,7 +456,7 @@ def test_select_legal_values(self):
     @skip_without_tool("column_param")
     def test_column_legal_values(self):
         history_id = self.dataset_populator.new_history()
-        new_dataset1 = self.dataset_populator.new_dataset(history_id, content='#col1\tcol2')
+        new_dataset1 = self.dataset_populator.new_dataset(history_id, content='#col1\tcol2', wait=True)
```

Comment:
This wasn't right I don't think - you made it so the test is no longer verifying the job is prevented from running when the column value is insecure and would cause a security problem. I've fixed the test case I think and pushed it to your branch - hope that is okay.

Reply:
Thanks.